sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/Processor/PartitionLoadBalancerTests.cs

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Azure.Messaging.EventHubs.Tests;
+using System.Collections.Generic;
+
+namespace Azure.Messaging.EventHubs.Processor.Tests
```

Comment:
This is a thing of beauty.    With the load balancer able to be independently tested, do you folks think that we can trim some things out of the processor live tests?   The load balancing items are particularly cranky around timing and also take longer than is ideal to execute due to the load balancing cycles.

//cc: @christothes , @kinelski, @ShivangiReja

Reply:
created #9215 to track this.

Reply:
Thank you.... that is very much appreciated.